piicatcher/scanner.py

```diff
@@ -65,6 +64,8 @@ def scan(self, text):
 
 
 class ColumnNameScanner(Scanner):
+    def __init__(self, exclude_regex=()):
+        self.excluded_regex = re.compile(exclude_regex[0], re.IGNORECASE)   # [0] cause the option comes as tuple
```

Comment:
Why `excluded` and not `exclude`? Isnt that inconsistent with the name used before?

Reply:
it would be reassigning the variable with a totally different thing.
Like earlier it's a tuple and now it would be a regex match. 
Not sure if reassigning the var to itself with a different object type is a good pattern or not. Hence changed the LHS var name.
If it's okay, can do that as well.

Reply:
Renamed to `_exclude_regex`